network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/hero.html

```diff
@@ -11,8 +11,9 @@
     <div class="tw-relative tw-space-y-5 medium:tw-space-y-6 tw-flex tw-flex-col tw-justify-center tw-items-start">
       <h1 class="tw-mb-0 tw-leading-none tw-font-bold tw-text-4xl medium:tw-text-[3.25rem]">{% trans "Does this button work?" %}</h1>
       <p class="tw-body-large large:tw-text-2xl">
+        <span class="tw-font-bold">{% trans "Investigating YouTube’s ineffective user controls" %}</span><br/>
         {% blocktrans trimmed %}
-          <span class="tw-font-bold">Investigating YouTube's ineffective user controls</span> powered by 22,722 volunteers, Mozilla scrutinized YouTube to determine how much control people actually have over the platform’s recommendation algorithm.
+          Powered by 22,722 volunteers, Mozilla scrutinized YouTube to determine how much control people actually have over the platform’s recommendation algorithm.
```

Comment:
This was treated as a single sentence, while it shouldn’t be the case (as per copy deck & redpen)